pysph/examples/rigid_body/rigid_block_in_tank.py

```diff
@@ -1,191 +1,250 @@
-"""A block of rigid solid falling on a tank of water. (6 minutes)
+"""A block of rigid solid falling on wood in a tank of water.
```

Comment:
Can you please retain and fix the original simpler example and add your new one as a separate example?  Thanks.